lib/splitio-api-binding.js

```diff
@@ -99,4 +99,21 @@ class SplitioApiBinding {
   }
 }
 
-module.exports = SplitioApiBinding
+function getSegmentsInUse ({ splitConditions }) {
```

Comment:
will `getSegmentNamesInUse` sound clearer since it is just returning the names of the segments, not the full data of segments